src/bridge/bridgemain.h

```diff
@@ -757,6 +757,12 @@ typedef struct
 
 typedef struct
 {
+    NTSTATUS code;

```

Comment:
If you use DWORD here just like `LASTERROR` it will not display the `ffffffff....`

Reply:
Indeed! I think I'll refrain from asking why this is... fixed regardless.